docs_user/modules/proc_adopting-the-identity-service.adoc

```diff
@@ -27,10 +26,10 @@ EOF
 
 .Procedure
 
-. Patch `OpenStackControlPlane` to deploy {identity_service}:
+. Patch the `OpenStackControlPlane` CR to deploy the {identity_service}:
```

Comment:
@xek In step 1, is the `openstack_control_plane.yaml` file being updated?

Reply:
It's the CR that's being updated, so the charge is correct